full-deps/fulldepsspack/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM 	   dealii/ubuntu16
+
+MAINTAINER luca.heltai@gmail.com
+
+USER 	   root
+
+# general environment for docker
+ENV        DEBIAN_FRONTEND=noninteractive \
+           SPACK_ROOT=/usr/local \
+           FORCE_UNSAFE_CONFIGURE=1
+
+# install minimal spack dependencies
+RUN        apt-get update \
+           && apt-get install -y --no-install-recommends \
+              autoconf \
+              ca-certificates \
+              curl \
+              environment-modules \
+              git \
+              build-essential \
+	      gfortran \
+              python \
+              nano \
+              unzip \
+           && rm -rf /var/lib/apt/lists/*
+
+# install spack
+RUN        curl -s -L https://api.github.com/repos/llnl/spack/tarball \
```

Comment:
I think that the "ownership" of spack has moved from `llnl` to `spack`, so this should be `https://api.github.com/repos/spack/spack/tarball`. @davydden Do you agree with this?

Reply:
yes, absolutely. It's official Github is https://github.com/spack/spack , that happened like a year ago.